apps/website/src/examples/Dialog.dismissible.tsx

```diff
@@ -0,0 +1,43 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import * as React from 'react';
+import {
+  Modal,
+  Button,
+  ModalContent,
+  ModalButtonBar,
+  LabeledInput,
+  LabeledTextarea,
+} from '@itwin/itwinui-react';
+
+export default () => {
+  const [isModalOpen, setIsModalOpen] = React.useState(true);
+  const closeModal = () => setIsModalOpen(false);
+
+  return (
+    <>
+      <Button styleType='high-visibility' onClick={() => setIsModalOpen(true)}>
+        Open dismissible dialog
+      </Button>
+      <Modal
```

Comment:
maybe most of the examples should use `Dialog`, not `Modal`, because it is more flexible and supports all use cases.

and then we can show `Modal` in just one example, saying that it is a "convenience" wrapper over `Dialog`.

@gretanausedaite thoughts?

Reply:
I have changes local that uses `Dialog` in every demo except for`Dialog.fullpage.tsx` (unsure how to achieve) & `Dialog.modal.tsx`.  Should I push or wait for Greta's response?

Reply:
go ahead and push it, you can always adjust/revert based on Greta's response

Reply:
Pushed!

Reply:
I somehow missed that other examples were Modal and not Dialog. Good example of you see what you want to see.
Very much agree with this change.